packages/manager/apps/catalog/src/components/Filters/FilterItem.tsx

```diff
@@ -4,7 +4,10 @@ import { OsdsCheckbox } from '@ovhcloud/ods-components/checkbox/react';
 import { OsdsCheckboxButton } from '@ovhcloud/ods-components/checkbox-button/react';
 import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
 import { ODS_CHECKBOX_BUTTON_SIZE } from '@ovhcloud/ods-components/checkbox-button';
-import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_THEME_COLOR_HUE,
```

Comment:
I think you forgot to remove this unused import, would you mind to clean it up pretty please ? 
[Sonarcloud](https://sonarcloud.io/project/issues?resolved=false&pullRequest=10671&id=ovh_manager&open=AYz9O84usTMlDTLTl5Cw)